.gitignore

```diff
@@ -87,3 +87,6 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+# Jetbrains project files
+.idea/
```

Comment:
Neat, didn't know that.  
https://git-scm.com/docs/gitignore
Just had to add the personal ignores here:`$HOME/.config/git/ignore`